workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -228,8 +235,17 @@ def handle(self, *args, **options):
                         events_for_update[event.slug] = changes
                         print('Detected changes in {}'.format(event.slug))
 
+            except GithubException:
+                print('GitHub error when accessing {} repo'.format(event.slug))
+
+            except socket.timeout:
+                print('Timeout when accessing {} repo'.format(event.slug))
+
             except WrongWorkshopURL:
                 print('Wrong URL for {}'.format(event.slug))
 
             except requests.exceptions.RequestException:
                 print('Network error when accessing {}'.format(event.slug))
+
+            except Exception as e:
+                print('Unknown error ({}): {}'.format(event.slug, e))
```

Comment:
Yeah, I'll change to stderr but this mostly doesn't matter. The output will be logged by cronjob to some file.

We won't throw fire extinguishers at GitHub API if these errors happen, and they happen because:
1. event's URL point to wrong repository or the repo was removed (I believe that's the case for most of the errors), but anyway it's nothing to worry about
2. PyGithub has some issues with one event's repository - again, not our burden.